examples/02-modal-harmonic/06-cycles_to_failure.py

```diff
@@ -0,0 +1,118 @@
+"""
+DPF-Core Fatigue Engineering Simple Example
+~~~~~~~~~~~~~~~~~~~~
+This example shows how to generate and use a result file to calculate the
+cycles to failure result for a simple model.
+Material data is manually imported, Structural Steel from Ansys Mechanical:
+Youngs Modulus (youngsSteel)
+Poisson's Ratio (prxySteel)
+Cycles to Failure (sn_data)
+The first step is to generate a simple model with high stress and save the
+results .rst locally to myDir (default C:\temp).
+For this we use a short pyMapdl script
+The second step uses DPF-Core to generate the cycles to failure result.
+The locally saved rst is imported and plotted.
+Then the von Mises stress is generated and plotted with DPF operators.
+The python package numpy is then used to interpolate the cycles to failure values.
+The nodal von Mises equivalent stress value is used in the interpolation.
+Note the cycles to failure data needs to be manipulated to use numpy interpolation.
+An empty field is created and filled with the resulting cycles to failure values.
+Cycles to failure result plotted.
+The cycles to failure result is the (interpolated) negative of the stress result.
+The higher the stress result, the lower the number of cycles to failure.
+"""
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+import numpy as np
+
+
+# The first step is to generate a simple model with high stress
+
+# # Material parameters from Ansys Mechanical Structural Steel
+youngsSteel = 200e9
+prxySteel = 0.3
+sn_data = np.empty((11, 2))  # initialize empty np matrix
+sn_data[:, 0] = [10, 20, 50, 100, 200, 2000, 10000, 20000, 1e5, 2e5, 1e6]
+sn_data[:, 1] = [3.999e9, 2.8327e9, 1.896e9, 1.413e9, 1.069e9, 4.41e8, 2.62e8, 2.14e8, 1.38e8,
+                 1.14e8, 8.62e7]
+
+###############################################################################
+
+# This .rst file is here already available but can be obtained using the short pyMapdl code below:
```

Comment:
This .rst file is already available, but can be obtained using the short pyMAPDL code below: